code/game/atoms.dm

```diff
@@ -2177,3 +2177,12 @@
 	var/mutable_appearance/glow_appearance = new(glow)
 	add_overlay(glow_appearance)
 	LAZYADD(update_overlays_on_z, glow_appearance)
+
+/atom/proc/SpinAnimation(speed = 1 SECONDS, loops = -1, clockwise = 1, segments = 3, parallel = TRUE)
```

Comment:
An autodoc here would be nice, even if it speaks mostly for itself, it's still a pretty global proc